### DISCUSSION TITLE
Suggested Edits Bugs and Feedback

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#2809] [General]
We've finally started rolling out Suggested Edits to communities who have requested it (if you're interested, please let us know by writing to hello@pubpub.org with the url of the community). We're using this thread to collect any bugs or feedback for the feature. Thanks very much for your help testing!

A few notes/caveats:
- We expect some amount of weirdness, so please do report any issues if you see them. In your reports, please include a link to the Pub, and a step-by-step recap of exactly what you saw, with screenshots if possible.
- Suggested Edits does not work for math nodes, sadly. We don't currently have a plan to fix this, since the issue lies in an upstream plugin we're using, but hope that we'll be able to over time.
- We're aware that tables are a little wonky, requiring accepting of individual cells, for example. We're strategizing how to improve this, but don't have a fix at this time.
- Finally, we know that currently you can't add comments on suggested ranges, and we're working to figure out how to enable this.

**Comment by mustafa0x** [comment]
Nice work! Is there a screenshot or any way to quickly evaluate this feature (eg storybook)? Thanks!
